lending-pool/src/auctions/bad_debt_auction.rs

```diff
@@ -101,8 +101,8 @@ pub fn fill_bad_debt_auction(
     if !backstop_state.positions.liabilities.is_empty() {
         let pool_backstop_data = backstop_client.pool_data(&e.current_contract_address());
         let threshold = calc_pool_backstop_threshold(&pool_backstop_data);
-        if threshold < 0_0000100 {
-            // ~10% of threshold
+        if threshold < 0_0000500 {
+            // ~5% of threshold
```

Comment:
The `calc_pool_backstop_threshold` takes a shortcut when calculating the percentage to ensure the result fits using basic i128 fixed point math.

From the function documentation:
```
/// Calculate the threshold for the pool's backstop balance
///
/// Returns the threshold as a percentage^5 in SCALAR_7 points such that SCALAR_7 = 100%
/// NOTE: The result is the percentage^5 to simplify the calculation of the pools product constant.
///       Some useful results:
///         - greater than 1 = 100+%
///         - 1_0000000 = 100%
///         - 0_0000100 = ~10%
///         - 0_0000003 = ~5%
///         - 0_0000000 = ~0-4%
```
We should be ok if the `threshold <= 0_0000003` to proceed.

We should also update the unit tests in both `bad_debt_auction.rs` and `status.rs` for that function to insure it behaves appropriately around the threshold chosen.